andela/travel_tool_front
# #160370620 Make multiple requests

#### What does this PR do?
This PR contains details for creating multiple requests by the user

#### Description of Task to be completed?
- A user should be able to see extra input fields when they click on multi-request
- A user should see new fields when they click on add another field
- Users should be able to post all request made to the the API
- Users should be able to delete input fields whilst making requests

#### How should this be manually tested?
- Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
- Switch to the repo directory - `cd travel_tool_front`
- Check out this branch - `git checkout ft-multitrip-modal-travel-request-160370620`
- Install dependencies and start the server  - `make start`
- Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/46)
- Navigate to `localhost:3000` in a browser
- Create a new travel request
- From the Modal that appears, select `Multi City Trip`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#[160370620](https://www.pivotaltracker.com/story/show/160370620)

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-09-15 at 2 58 20 pm" src="https://user-images.githubusercontent.com/22764463/45687081-aaf32880-bb45-11e8-97cd-baa91bcfdba2.png">



#### Questions:
